src/heretic/p_lights.c

```diff
@@ -52,6 +53,11 @@ void T_LightFlash(thinker_t *thinker)
         flash->count = (P_Random() & flash->maxtime) + 1;
     }
 
+    // [crispy] A11Y
+    if (a11y_sector_lighting)
+	flash->sector->rlightlevel = flash->sector->lightlevel;
+    else
+	flash->sector->rlightlevel = flash->maxlight;
```

Comment:
Required to restore maxlight for those sectors after loading a save file.

Reply:
Speaking of savegames, I had to add some code to `doom/p_extsave.c` to make sure that both values are restored for sectors for which `sector->lightlevel` and `sector->rlightlevel` differ. I don't see any such code in your PR. Does this mean that the code I added for Doom becomes unnecessary with the changes you introduced here?

Reply:
I experimented with storing, but eventually undid that. Since all the flash-lights are thinkers and they are already stored, I took the maxlight level from there to restore it on startup/load of a game. Without further testing, I presume the same could be done in Doom aswell without separate handling when storing the lightlevel. 

In fact, the code in p_extsave.c doesn't appear to prevent inconsitent light-levels when changing the flickering-option between saves. You can verify this by:

- Enable Sector Light Flickering in Doom.
- Go up to a flickering light and save when the lightlevel is minimum/off (might require some timing)
- Change Setup to Disable Sector Light Flickering.
- Load the previous save

Expected Result:
Light should be a static maxlight

Actual Result:
Light remains static on last saved flickering value. 

However, there is still one finding as I undid also the unarchiving of the rlightlevel by mistake. I have corrected that now. This is required to ensure that changes to lights ( e. g. caused by triggers) are correctly restored when loading a save.

Reply:
> Alright, I see now. Seems you have got better insight into the topic than I had when implementing it for Doom. 😁 Could you please do me a favour and fix all the things you learned here for the Doom code base as well? I guess the `P_WriteRLightlevel()` and `P_ReadRLightlevel()` functions in `doom/p_extsaveg.c` could be turned into NOPs then.

No worries! Like mentioned in the issue post, this also caused me quite some headache when doing the first proposal. 😀 
Sure, I will take a look at doom and open a seperate pull-request!

Reply:
Thank you!